### ISSUE TITLE
[Accessibility : Focus & sequence] No focus for Action dropdown in network configuration 

### ISSUE THREAD

**Issue by atreyeemukhopadhyay** [kimchi-project/ginger#282]
While navigating through the network configuration section  using keyboard action dropdown is not getting focused.

![network configuration action](https://cloud.githubusercontent.com/assets/13199397/14106597/dd570b40-f5cf-11e5-847d-76de936c6538.jpg)
